### DISCUSSION TITLE
posts/2024-09-10-gaps-risks-science/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#19] [Announcements]
# posts/2024-09-10-gaps-risks-science/

Recognizing gaps in how AI is applied to scientific research can prevent potential shortcomings and risks.

https://rachel.fast.ai/posts/2024-09-10-gaps-risks-science/

**Comment by sakshiepathak** [comment]
Hi Rachel, 

I love your blog. I also love to learn new things. 
I have a bachelor's in CS but currently work as a Product Manager in a tech company. I especially liked 'There's no such thing as not a math person' article of yours. I must've read it around 10 times. 

I realised I have put myself in a box and have been afraid of coding and math. To change this, I want to learn math and coding. (Preferably in the US, Indian education quality is absolute garbage)

However, I don't know which programs to apply to. I wanted to ask you for course recommendations. I neither have much coding experience (have only learnt from online courses) nor do I come from a prestigious university. Oh, also, I live in India. 

Also, what do you know about Computational Science & Engg? I have been exploring these programs and liked them.

I would really appreciate any guidance/tips you could give me. 

Thanks in advance,
Sakshie

**Reply by racheltho** [reply]
Hi Sakshie, thanks for messaging.  I am glad that the 'There's no such thing as not a math person' article was helpful!  Regarding education options, there are so many different options, all of which depend on a range of factors-- what your long-term goals are, costs and scholarships, where you are accepted, etc.  It is impossible to make a blanket recommendation of course recommendations.  

I do think it may be useful to:

- continue taking online courses (which can provide helpful background even when you pursue more formal/traditional programs)
- blog or build a side project about what you are learning
- seek out others who have made the career transition you hope for in order to learn from their experiences & advice
- seek out a community (either in person through meetups or online) of others who are working on learning the same skills that you are interested in.  Having peers to study with can be helpful and provide accountability.
